main/src/ca/uwaterloo/flix/language/phase/TypeReconstruction.scala

```diff
@@ -417,6 +417,11 @@ object TypeReconstruction {
       val eff = Type.mkUnion(eff1 :: es.map(_.eff), loc)
       TypedAst.Expr.Do(op, es, tpe, eff, loc)
 
+    case KindedAst.Expr.InvokeMethod2(exp, name, exps, tvar, evar, loc) =>
+      val e = visitExp(exp)
+      val es = exps.map(visitExp(_))
```

Comment:
visitExp(_) -> visitExp

Reply:
(You can fix this later)

Reply:
Thank you. This should be fixed in #7762.